### ISSUE TITLE
`list-prs-for-file`: PR list appears intermittently and persists when selecting unrelated file

### ISSUE THREAD

**Issue by aidenmitchell** [refined-github/refined-github#6849]
### Description

PR doesn't appear:
![image](https://github.com/refined-github/refined-github/assets/30846409/dde6ca8d-1fe3-4301-bd6c-217847172dd5)

Appears after a page refresh:
![image](https://github.com/refined-github/refined-github/assets/30846409/9dcffb5a-b1b3-4372-9c82-26ad62aaa9de)

Persists on unrelated files:
![image](https://github.com/refined-github/refined-github/assets/30846409/1d4dd88d-98b3-4671-b47a-8c93d044d17d)

Console log:
```
[Error] Refused to execute a script because its hash, its nonce, or 'unsafe-inline' does not appear in the script-src directive of the Content Security Policy. (attachment_office365_image.yml, line 0, x2)
[Info] Successfully preconnected to https://github.githubassets.com/
[Info] Successfully preconnected to https://avatars.githubusercontent.com/
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office365_image.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendEvent (environment-509b58e05b9f.js:5:1633)
	p (environment-509b58e05b9f.js:1:8898)
	(anonymous function) (ui_packages_alive_alive_ts-ui_packages_alive_connect-alive-subscription_ts-app_assets_modules-face4e-ab9ea68fa960.js:4:175961)
	ut (react-lib-210c4b5934c3.js:25:96193)
	ip (react-lib-210c4b5934c3.js:25:114225)
	u8 (react-lib-210c4b5934c3.js:25:108399)
	u8
	av (react-lib-210c4b5934c3.js:25:47337)
	id (react-lib-210c4b5934c3.js:25:113073)
	ic (react-lib-210c4b5934c3.js:25:112039)
	u1 (react-lib-210c4b5934c3.js:25:106985)
	u1
	x (react-lib-210c4b5934c3.js:49:1375)
	R (react-lib-210c4b5934c3.js:49:1905)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office365_image.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendPageView (environment-509b58e05b9f.js:5:1552)
	f (environment-509b58e05b9f.js:1:8693)
	(anonymous function) (ui_packages_react-core_register-app_ts-c9d112423e9d.js:1:6042)
	ut (react-lib-210c4b5934c3.js:25:96193)
	ip (react-lib-210c4b5934c3.js:25:114225)
	u8 (react-lib-210c4b5934c3.js:25:108399)
	u8
	av (react-lib-210c4b5934c3.js:25:47337)
	id (react-lib-210c4b5934c3.js:25:113073)
	ic (react-lib-210c4b5934c3.js:25:112039)
	u1 (react-lib-210c4b5934c3.js:25:106985)
	u1
	x (react-lib-210c4b5934c3.js:49:1375)
	R (react-lib-210c4b5934c3.js:49:1905)
[Error] Failed to load resource: the server responded with a status of 404 () (23.8.3.css, line 0)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office365_image.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendPageView (environment-509b58e05b9f.js:5:1552)
	f (environment-509b58e05b9f.js:1:8693)
	(anonymous function) (behaviors-f2d6c08163e6.js:8:53654)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office365_image.yml from loading a resource from https://api.github.com/_private/browser/stats (ui_packages_soft-nav_soft-nav_ts-df17d5597d8f.js, line 1)
[Error] Beacon API cannot load https://api.github.com/_private/browser/stats. Resource blocked by content blocker
	E (ui_packages_soft-nav_soft-nav_ts-df17d5597d8f.js:1:9227)
	f (ui_packages_soft-nav_soft-nav_ts-df17d5597d8f.js:1:8874)
	(anonymous function) (vendors-node_modules_stacktrace-parser_dist_stack-trace-parser_esm_js-node_modules_github_bro-a4c183-18bf85b8e9f4.js:1:10432)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office365_image.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendEvent (environment-509b58e05b9f.js:5:1633)
	p (environment-509b58e05b9f.js:1:8898)
	rk (behaviors-f2d6c08163e6.js:8:46596)
	(anonymous function) (vendors-node_modules_stacktrace-parser_dist_stack-trace-parser_esm_js-node_modules_github_bro-a4c183-18bf85b8e9f4.js:1:10432)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office365_image.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendEvent (environment-509b58e05b9f.js:5:1633)
	p (environment-509b58e05b9f.js:1:8898)
	(anonymous function) (app_assets_modules_github_command-palette_copy_ts-app_assets_modules_react-shared_hooks_use-v-bce6ee-88c2810e530f.js:5:1889)
	(anonymous function) (ui_packages_alive_alive_ts-ui_packages_alive_connect-alive-subscription_ts-app_assets_modules-face4e-ab9ea68fa960.js:1:22312)
	(anonymous function) (ui_packages_alive_alive_ts-ui_packages_alive_connect-alive-subscription_ts-app_assets_modules-face4e-ab9ea68fa960.js:4:216706)
	onClick (vendors-node_modules_primer_react_lib-esm_TreeView_TreeView_js-d86950fa1004.js:1:11099)
	eH (react-lib-210c4b5934c3.js:25:16717)
	eX (react-lib-210c4b5934c3.js:25:16871)
	eG (react-lib-210c4b5934c3.js:25:16928)
	rw (react-lib-210c4b5934c3.js:25:38006)
	rS (react-lib-210c4b5934c3.js:25:38422)
	(anonymous function) (react-lib-210c4b5934c3.js:25:43321)
	u5 (react-lib-210c4b5934c3.js:25:108731)
	eV (react-lib-210c4b5934c3.js:25:15848)
	rP (react-lib-210c4b5934c3.js:25:39713)
	tJ (react-lib-210c4b5934c3.js:25:24064)
	tG (react-lib-210c4b5934c3.js:25:23848)
	tG
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office_remote_doc_template.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendEvent (environment-509b58e05b9f.js:5:1633)
	p (environment-509b58e05b9f.js:1:8898)
	(anonymous function) (ui_packages_alive_alive_ts-ui_packages_alive_connect-alive-subscription_ts-app_assets_modules-face4e-ab9ea68fa960.js:4:175961)
	ut (react-lib-210c4b5934c3.js:25:96193)
	ip (react-lib-210c4b5934c3.js:25:114225)
	(anonymous function) (react-lib-210c4b5934c3.js:25:112493)
	x (react-lib-210c4b5934c3.js:49:1375)
	R (react-lib-210c4b5934c3.js:49:1905)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office_remote_doc_template.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendPageView (environment-509b58e05b9f.js:5:1552)
	f (environment-509b58e05b9f.js:1:8693)
	(anonymous function) (ui_packages_react-core_register-app_ts-c9d112423e9d.js:1:6042)
	ut (react-lib-210c4b5934c3.js:25:96193)
	ip (react-lib-210c4b5934c3.js:25:114225)
	(anonymous function) (react-lib-210c4b5934c3.js:25:112493)
	x (react-lib-210c4b5934c3.js:49:1375)
	R (react-lib-210c4b5934c3.js:49:1905)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office_remote_doc_template.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendPageView (environment-509b58e05b9f.js:5:1552)
	f (environment-509b58e05b9f.js:1:8693)
	(anonymous function) (behaviors-f2d6c08163e6.js:8:53617)
	dispatchEvent
	p (ui_packages_soft-nav_soft-nav_ts-df17d5597d8f.js:1:5178)
	k (ui_packages_react-core_register-app_ts-c9d112423e9d.js:1:6256)
	(anonymous function) (ui_packages_react-core_register-app_ts-c9d112423e9d.js:1:6176)
	ut (react-lib-210c4b5934c3.js:25:96193)
	ip (react-lib-210c4b5934c3.js:25:114225)
	(anonymous function) (react-lib-210c4b5934c3.js:25:112493)
	x (react-lib-210c4b5934c3.js:49:1375)
	R (react-lib-210c4b5934c3.js:49:1905)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office_remote_doc_template.yml from loading a resource from https://api.github.com/_private/browser/stats (ui_packages_soft-nav_soft-nav_ts-df17d5597d8f.js, line 1)
[Error] Beacon API cannot load https://api.github.com/_private/browser/stats. Resource blocked by content blocker
	E (ui_packages_soft-nav_soft-nav_ts-df17d5597d8f.js:1:9227)
	f (ui_packages_soft-nav_soft-nav_ts-df17d5597d8f.js:1:8874)
	(anonymous function) (vendors-node_modules_stacktrace-parser_dist_stack-trace-parser_esm_js-node_modules_github_bro-a4c183-18bf85b8e9f4.js:1:10432)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office_remote_doc_template.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendEvent (environment-509b58e05b9f.js:5:1633)
	p (environment-509b58e05b9f.js:1:8898)
	(anonymous function) (app_assets_modules_github_command-palette_copy_ts-app_assets_modules_react-shared_hooks_use-v-bce6ee-88c2810e530f.js:5:1889)
	(anonymous function) (ui_packages_alive_alive_ts-ui_packages_alive_connect-alive-subscription_ts-app_assets_modules-face4e-ab9ea68fa960.js:1:22312)
	(anonymous function) (ui_packages_alive_alive_ts-ui_packages_alive_connect-alive-subscription_ts-app_assets_modules-face4e-ab9ea68fa960.js:4:216706)
	onClick (vendors-node_modules_primer_react_lib-esm_TreeView_TreeView_js-d86950fa1004.js:1:11099)
	eH (react-lib-210c4b5934c3.js:25:16717)
	eX (react-lib-210c4b5934c3.js:25:16871)
	eG (react-lib-210c4b5934c3.js:25:16928)
	rw (react-lib-210c4b5934c3.js:25:38006)
	rS (react-lib-210c4b5934c3.js:25:38422)
	(anonymous function) (react-lib-210c4b5934c3.js:25:43321)
	u5 (react-lib-210c4b5934c3.js:25:108731)
	eV (react-lib-210c4b5934c3.js:25:15848)
	rP (react-lib-210c4b5934c3.js:25:39713)
	tJ (react-lib-210c4b5934c3.js:25:24064)
	tG (react-lib-210c4b5934c3.js:25:23848)
	tG
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office365_image.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendEvent (environment-509b58e05b9f.js:5:1633)
	p (environment-509b58e05b9f.js:1:8898)
	(anonymous function) (ui_packages_alive_alive_ts-ui_packages_alive_connect-alive-subscription_ts-app_assets_modules-face4e-ab9ea68fa960.js:4:175961)
	ut (react-lib-210c4b5934c3.js:25:96193)
	ip (react-lib-210c4b5934c3.js:25:114225)
	u1 (react-lib-210c4b5934c3.js:25:106324)
	u1
	x (react-lib-210c4b5934c3.js:49:1375)
	R (react-lib-210c4b5934c3.js:49:1905)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office365_image.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendPageView (environment-509b58e05b9f.js:5:1552)
	f (environment-509b58e05b9f.js:1:8693)
	(anonymous function) (ui_packages_react-core_register-app_ts-c9d112423e9d.js:1:6042)
	ut (react-lib-210c4b5934c3.js:25:96193)
	ip (react-lib-210c4b5934c3.js:25:114225)
	u1 (react-lib-210c4b5934c3.js:25:106324)
	u1
	x (react-lib-210c4b5934c3.js:49:1375)
	R (react-lib-210c4b5934c3.js:49:1905)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office365_image.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendPageView (environment-509b58e05b9f.js:5:1552)
	f (environment-509b58e05b9f.js:1:8693)
	(anonymous function) (behaviors-f2d6c08163e6.js:8:53617)
	dispatchEvent
	p (ui_packages_soft-nav_soft-nav_ts-df17d5597d8f.js:1:5178)
	k (ui_packages_react-core_register-app_ts-c9d112423e9d.js:1:6256)
	(anonymous function) (ui_packages_react-core_register-app_ts-c9d112423e9d.js:1:6176)
	ut (react-lib-210c4b5934c3.js:25:96193)
	ip (react-lib-210c4b5934c3.js:25:114225)
	u1 (react-lib-210c4b5934c3.js:25:106324)
	u1
	x (react-lib-210c4b5934c3.js:49:1375)
	R (react-lib-210c4b5934c3.js:49:1905)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office365_image.yml from loading a resource from https://api.github.com/_private/browser/stats (ui_packages_soft-nav_soft-nav_ts-df17d5597d8f.js, line 1)
[Error] Beacon API cannot load https://api.github.com/_private/browser/stats. Resource blocked by content blocker
	E (ui_packages_soft-nav_soft-nav_ts-df17d5597d8f.js:1:9227)
	f (ui_packages_soft-nav_soft-nav_ts-df17d5597d8f.js:1:8874)
	(anonymous function) (vendors-node_modules_stacktrace-parser_dist_stack-trace-parser_esm_js-node_modules_github_bro-a4c183-18bf85b8e9f4.js:1:10432)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office365_image.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendEvent (environment-509b58e05b9f.js:5:1633)
	p (environment-509b58e05b9f.js:1:8898)
	(anonymous function) (app_assets_modules_github_command-palette_copy_ts-app_assets_modules_react-shared_hooks_use-v-bce6ee-88c2810e530f.js:5:1889)
	(anonymous function) (ui_packages_alive_alive_ts-ui_packages_alive_connect-alive-subscription_ts-app_assets_modules-face4e-ab9ea68fa960.js:1:22312)
	(anonymous function) (ui_packages_alive_alive_ts-ui_packages_alive_connect-alive-subscription_ts-app_assets_modules-face4e-ab9ea68fa960.js:4:216706)
	onClick (vendors-node_modules_primer_react_lib-esm_TreeView_TreeView_js-d86950fa1004.js:1:11099)
	eH (react-lib-210c4b5934c3.js:25:16717)
	eX (react-lib-210c4b5934c3.js:25:16871)
	eG (react-lib-210c4b5934c3.js:25:16928)
	rw (react-lib-210c4b5934c3.js:25:38006)
	rS (react-lib-210c4b5934c3.js:25:38422)
	(anonymous function) (react-lib-210c4b5934c3.js:25:43321)
	u5 (react-lib-210c4b5934c3.js:25:108731)
	eV (react-lib-210c4b5934c3.js:25:15848)
	rP (react-lib-210c4b5934c3.js:25:39713)
	tJ (react-lib-210c4b5934c3.js:25:24064)
	tG (react-lib-210c4b5934c3.js:25:23848)
	tG
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_pdf_with_google_ae_redirect.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendEvent (environment-509b58e05b9f.js:5:1633)
	p (environment-509b58e05b9f.js:1:8898)
	(anonymous function) (ui_packages_alive_alive_ts-ui_packages_alive_connect-alive-subscription_ts-app_assets_modules-face4e-ab9ea68fa960.js:4:175961)
	ut (react-lib-210c4b5934c3.js:25:96193)
	ip (react-lib-210c4b5934c3.js:25:114225)
	(anonymous function) (react-lib-210c4b5934c3.js:25:112493)
	x (react-lib-210c4b5934c3.js:49:1375)
	R (react-lib-210c4b5934c3.js:49:1905)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_pdf_with_google_ae_redirect.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendPageView (environment-509b58e05b9f.js:5:1552)
	f (environment-509b58e05b9f.js:1:8693)
	(anonymous function) (ui_packages_react-core_register-app_ts-c9d112423e9d.js:1:6042)
	ut (react-lib-210c4b5934c3.js:25:96193)
	ip (react-lib-210c4b5934c3.js:25:114225)
	(anonymous function) (react-lib-210c4b5934c3.js:25:112493)
	x (react-lib-210c4b5934c3.js:49:1375)
	R (react-lib-210c4b5934c3.js:49:1905)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_pdf_with_google_ae_redirect.yml from loading a resource from https://collector.github.com/github/collect (environment-509b58e05b9f.js, line 5)
[Error] Beacon API cannot load https://collector.github.com/github/collect. Resource blocked by content blocker
	send (environment-509b58e05b9f.js:5:1821)
	sendPageView (environment-509b58e05b9f.js:5:1552)
	f (environment-509b58e05b9f.js:1:8693)
	(anonymous function) (behaviors-f2d6c08163e6.js:8:53617)
	dispatchEvent
	p (ui_packages_soft-nav_soft-nav_ts-df17d5597d8f.js:1:5178)
	k (ui_packages_react-core_register-app_ts-c9d112423e9d.js:1:6256)
	(anonymous function) (ui_packages_react-core_register-app_ts-c9d112423e9d.js:1:6176)
	ut (react-lib-210c4b5934c3.js:25:96193)
	ip (react-lib-210c4b5934c3.js:25:114225)
	(anonymous function) (react-lib-210c4b5934c3.js:25:112493)
	x (react-lib-210c4b5934c3.js:49:1375)
	R (react-lib-210c4b5934c3.js:49:1905)
[Info] Content blocker prevented frame displaying https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_pdf_with_google_ae_redirect.yml from loading a resource from https://api.github.com/_private/browser/stats (ui_packages_soft-nav_soft-nav_ts-df17d5597d8f.js, line 1)
[Error] Beacon API cannot load https://api.github.com/_private/browser/stats. Resource blocked by content blocker
	E (ui_packages_soft-nav_soft-nav_ts-df17d5597d8f.js:1:9227)
	f (ui_packages_soft-nav_soft-nav_ts-df17d5597d8f.js:1:8874)
	(anonymous function) (vendors-node_modules_stacktrace-parser_dist_stack-trace-parser_esm_js-node_modules_github_bro-a4c183-18bf85b8e9f4.js:1:10432)
```

### How to replicate the issue + URL

https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_office365_image.yml

### Extension version

23.8.3

### Browser(s) used

Safari

**Comment by fregante** [comment]
Duplicate of #6554
